### ISSUE TITLE
Start from launcher should show instructions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [pkern/kitcard-reader#2]
```
What steps will reproduce the problem?
1. Start the application from the launcher.

What is the expected output? What do you see instead?
Expected: "Please hold the KITCard no further than 2cm away from the device."
Instead: "0,00 € balance" is displayed
```

Original issue reported on code.google.com by `philipp.kern` on 4 Sep 2012 at 9:05

**Comment by GoogleCodeExporter** [comment]
```
Implemented in revision 3b5ffd9ee7da. Might still need some UI polishing (going 
back from ScanActivity should go back to the launcher; text size / placement 
fixes).
```

Original comment by `philipp.kern` on 24 Sep 2012 at 7:43
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `philipp.kern` on 7 Oct 2012 at 8:20
- Changed state: **Verified**
